packages/jupyter-ai/src/hooks/use-copy.ts

```diff
@@ -29,6 +30,14 @@ export type UseCopyReturn = {
    * the `useCopy()` hook.
    */
   copyLabel: string;
+  /**
+   * Boolean flag that indicates whether the copy functionality is disabled or not.
+   * This helps to disable the copy action button. The value is based on the presence
+   * of `navigator.clipboard`. If `navigator.clipboard` is unavailable, the copy
+   * functionality is considered disabled, and the `copyStatus` will be set to
+   * either `CopyStatus.None` or `CopyStatus.Disabled`.
+   */
+  isCopyDisabled: boolean;
```

Comment:
I don't think adding `isCopyDisabled` is necessary here, since consumers can already check if the status is disabled by using this condition:


```tsx
disabled={copyStatus === CopyStatus.Disabled}
```

Generally, we should avoid having 2 sources of truth for the same thing, since a future change might accidentally cause the two to differ.